### ISSUE TITLE
[kugo] Missing camera library on Oreo 8.1 compile

### ISSUE THREAD

**Issue by chris42** [sonyxperiadev/bug_tracker#64]
**Platform:** Loire 
**Device:** Kugo (Xperia X Compact)
**Kernel version:** 4.4
**Android version:** 8.1

**Description**
Following the build guide to Oreo 8.1, compile stops with error right in the beginning due to missing camera dependency.

**Symptoms**
Error messages:
```
hardware/qcom/camera/QCamera2/Android.mk: error: camera.msm8952 (SHARED_LIBRARIES android-arm) missing libdualcameraddm (SHARED_LIBRARIES android-arm) 
You can set ALLOW_MISSING_DEPENDENCIES=true in your environment if this is intentional, but that may defer real problems until later in the build.
build/core/main.mk:728: error: exiting from previous errors.
12:41:14 ckati failed with: exit status 1
```

**How to reproduce**
Follow build guide for 8.1, Kernel 4.4 building for Xperia X Compact

**Comment by jerpelea** [comment]
the 8.1 build guide is work in progress and will be fixed soon

**Comment by chris42** [comment]
Thanks for the info. Could you please update here as soon as the changes are done, then I can test and close the issue.

**Comment by jerpelea** [comment]
the camera HAL has been fixed and the dependency is not need 
please note that camera will not work without a vendor released for Android 8.1 4.4_6.4_r1
